bpf/lib/srv6.h

```diff
@@ -133,6 +181,14 @@ srv6_encapsulation(struct __ctx_buff *ctx, int growth, __u16 new_payload_len,
 		.hop_limit   = IPDEFTTL,
 	};
 
+#ifdef ENABLE_SRV6_FLOW_LABEL_PORT
```

Comment:
In my opinion, we don't need this ifdef. We can set the flow label by default.

Reply:
I was simply cautious in proposing the change that will change the default behavior. If it can be accepted that flow label will be set by default, then indeed no parameter control is required for this feature. Will remove all related proposed changes around the compiler flag then

Reply:
I agree with @YutaroHayakawa. There is no need for any flag here. 

Having the FL of the outer IPv6 header set to 0 is a bug. We should fix it simply setting the FL value based on the hash.

Reply:
Yeah, in my understanding, there's no use case for the zero flow label. It just loses the scalability in the ECMP or RSS scenario. If someone complains about the behavior change, we can always introduce the opt-out flag later.

Reply:
in-sync. that will make the code change much smaller :) as most of the code in the commit was related to flag hanndling